pulsar/error.go

```diff
@@ -103,14 +107,8 @@ const (
 	ProducerClosed
 	// SchemaFailure means the payload could not be encoded using the Schema
 	SchemaFailure
-
-	// ReachMaxPendingOps means the pending operations in transaction_impl coordinator reach the maximum.
-	ReachMaxPendingOps
 	// InvalidStatus means the component status is not as expected.
 	InvalidStatus
-	// TransactionError means this is a transaction related error
-	TransactionError
-
```

Comment:
Just a note here: it's a breaking change. But these error codes were added in http://github.com/apache/pulsar-client-go/pull/953, which is not included in any official release yet. So it's okay.

Reply:
Yes, we don't need this exception for now. so you can delete it first.